cultivation/scripts/flashcore/cli/main.py

```
@@ -0,0 +1,223 @@
+
+@app.command()
+def vet(
+    check: bool = typer.Option(
+        False,
+        "--check",
+        help="Run in check-only mode without modifying files. Exits with 1 if changes are needed.",
+    ),
+    files: Optional[List[Path]] = typer.Argument(
+        None,
+        help="Optional list of files to process. If not provided, vets all files in the source directory.",
+    ),
```

Ruff (0.11.9)

Do not perform function call `typer.Argument` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)